draft-ietf-ace-coap-est-oscore.md

```diff
@@ -279,6 +279,26 @@ In addition, EST-oscore uses the same CoAP Content-Format identifiers when trans
 |       | application/csrattrs                           (res) |  285  |
 {: #table_mediatypes cols="l l" title="EST functions and the associated CoAP Content-Format identifiers"}
 
+Content-Format 281 MUST be supported by EST-oscore servers.
+Servers MAY also support Content-Format 287.
+It is up to the client to support only Content-Format 281, 287 or both.
+As indicated in {{Section 4.3 of RFC9148}}, the client will use a CoAP Accept Option in the request to express the preferred response Content-Format.
+If an Accept Option is not included in the request, the client is not expressing any preference and the server SHOULD choose format 281.
+
+The generated response for /skg and /skc requests contains two parts: certificate and the corresponding private key.
+{{Section 4.8 of RFC9148}} specifies that the private key in response to /skc request may be either an encrypted (PKCS #7) or unencrypted (PKCS #8) key, depending on whether the CSR request included SMIMECapabilities.
+
+Due to the use of OSCORE, which protects the communication between the EST client and the EST server end-to-end, it is possible to return the private key to /skc as an unencrypted PKCS #8 object (Content-Format identifier 284).
+Therefore, when making the CSR to /skc, the EST client MUST NOT include SMIMECapabilities.
+As a consequence, the response to /skc is an unencrypted PKCS #8 object.
+
+{{table_cft_skg_skc}} summarizes the Content-Format identifiers used in responses to /skg and /skc.
+
+| Function | Response, Part 1 | Response, Part 2 |
+| /skg     | 284              | 281              |
+| /skc     | 284              | 287              |
+{: #table_cft_skg_skc cols="l l" title="Response Content-Format identifiers for /skg and /skc"}
+
```

Comment:
Based on the comment above, I wonder if, consistently with Section 4.8 of RFC9148, the column "Response, part 1" for row "/skg" should actually be "284 or 280". At the moment, that row here is at least consistent with the "/skg" row of Table 3 in Section 4.3 of RFC 9148.

Reply:
Thanks Marco. I edited the text to read:

> Due to the use of OSCORE, which protects the communication between the EST client and the EST server end-to-end, it is possible to return the private key to /skc or /skg as an unencrypted PKCS #8 object (Content-Format identifier 284). Therefore, when making the CSR to /skc or /skg, the EST client MUST NOT include SMIMECapabilities. As a consequence, the private key part of the response to /skc or /skg is an unencrypted PKCS #8 object.

and kept the table as-is.